src/Altinn.Platform/Altinn.Platform.Authentication/Altinn.Platform.Authentication.Tests/Mocks/EnterpriseUserAuthenticationServiceMock.cs

```diff
@@ -72,15 +88,14 @@ public async Task<HttpResponseMessage> AuthenticateEnterpriseUser(EnterpriseUser
                     Content = new StringContent(credentialsJson.ToString(), Encoding.UTF8, "application/json")
                 };
 
-                var result = await _client.SendAsync(request).ConfigureAwait(false);
                 result.StatusCode = HttpStatusCode.NotFound;
                 return result;
             }
         }
 
-        private HttpContent GetEnterpriseUserContent()
+        private static HttpContent GetEnterpriseUserContent(string userName)
         {
-            string jsonString = "{\r\n\t\"UserID\": 1234,\r\n\t\"Username\": \"ValidUser\",\r\n\t\"SSN\": null,\r\n\t\"PartyID\": 0,\r\n\t\"AuthenticateResult\": 1,\r\n\t\"AuthenticationMethod\": 9,\r\n\t\"LockedOutDate\": \"1753-01-01T00:00:00\",\r\n\t\"SmsPinUpgraded\": false,\r\n\t\"IsTestUser\": false,\r\n\t\"IDPortenNameID\": null,\r\n\t\"IDPortenSessionIndex\": null\r\n}";
+            string jsonString = string.Format("{{\r\n\t\"UserID\": 1234,\r\n\t\"Username\": \"{0}\",\r\n\t\"SSN\": null,\r\n\t\"PartyID\": 0,\r\n\t\"AuthenticateResult\": 1,\r\n\t\"AuthenticationMethod\": 9,\r\n\t\"LockedOutDate\": \"1753-01-01T00:00:00\",\r\n\t\"SmsPinUpgraded\": false,\r\n\t\"IsTestUser\": false,\r\n\t\"IDPortenNameID\": null,\r\n\t\"IDPortenSessionIndex\": null\r\n}}", userName);
```

Comment:
Recommend using string interpolation
```suggestion
            string jsonString = $"{{\r\n\t\"UserID\": 1234,\r\n\t\"Username\": \"{userName}\",\r\n\t\"SSN\": null,\r\n\t\"PartyID\": 0,\r\n\t\"AuthenticateResult\": 1,\r\n\t\"AuthenticationMethod\": 9,\r\n\t\"LockedOutDate\": \"1753-01-01T00:00:00\",\r\n\t\"SmsPinUpgraded\": false,\r\n\t\"IsTestUser\": false,\r\n\t\"IDPortenNameID\": null,\r\n\t\"IDPortenSessionIndex\": null\r\n}}";
```

Reply:
Changed